src/settings/Settings.js

```diff
@@ -320,10 +320,15 @@ export const SETTINGS = {
         supportedLevels: ['room-device'],
         default: false,
     },
-    "alwaysInviteUnknownUsers": {
+    "RoomSubList.showEmpty": {
```

Comment:
This was recently [removed](https://github.com/matrix-org/matrix-react-sdk/commit/e08c70b82089299bbf736f92dafe6a8ec5daf773#diff-7343e79e45dd5943e6d8ef53ab1c231a) from experimental, so this merge doesn't seem quite right to me.

Reply:
Good catch. I wonder what I did wrong...